airbyte-integrations/connectors/source-mailchimp/source_mailchimp/streams.py

```diff
@@ -138,6 +138,43 @@ def path(self, **kwargs) -> str:
         return "lists"
 
 
+class ListMembers(IncrementalMailChimpStream):
+    cursor_field = "last_changed"
+    data_field = "members"
+
+    def read_records(self, sync_mode: SyncMode, cursor_field: List[str] = None, stream_slice: Mapping[str, Any] = None, stream_state: Mapping[str, Any] = None) -> Iterable[Mapping[str, Any]]:
+        lists_stream = Lists(**kwargs)        
+        for list_record in lists_stream.read_records(sync_mode=SyncMode.full_refresh):
+            list_id = list_record["id"]
+            yield from super().read_records(sync_mode=SyncMode.full_refresh, stream_slice={"list_id": list_id})
+
+    def path(self, stream_state: Mapping[str, Any] = None, stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None) -> str:
+        list_id = stream_slice["list_id"]
+        return f"lists/{list_id}/members"
+
+
+class GetMemberInfo(IncrementalMailChimpStream):
+    cursor_field = "last_changed"
+
+    def read_records(self, sync_mode: SyncMode, cursor_field: List[str] = None, stream_slice: Mapping[str, Any] = None, stream_state: Mapping[str, Any] = None) -> Iterable[Mapping[str, Any]]:
+        listmembers_stream = ListMembers(**kwargs)        
```

Comment:
Try explicitly passing in the authenticator: i.e listsmembers_stream = ListMembers(authenticator=self.authenticator)

Reply:
I get this 
`  GetMemberInfo(authenticator=authenticator)\nTypeError: Can't instantiate abstract class GetMemberInfo with abstract method data_field\n", "failure_type": "system_error"}}}`

Reply:
This error has to do with the fact that there's a missing data_field definition here. Add data_field to the Get Members Info class.

Reply:
Yes that solves the problem :>